src/p2pool_api.h

```diff
@@ -80,6 +81,9 @@ class p2pool_api
 	std::string m_apiPath;
 	std::string m_networkPath;
 	std::string m_poolPath;
+	std::string m_localPath;
+
+	bool m_localStats;
```

Comment:
This `m_localStats` is not used anywhere?

Reply:
Removed, push pending.